volumes/csi-wrapper/hack/azure/azure-file-StorageClass-for-peerpod.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: azure-file-storage
+provisioner: file.csi.azure.com
+parameters:
+  skuName: Standard_LRS
```

Comment:
please add one new parameter
```
peerpod: "true"
```
It is need at https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/volumes/csi-wrapper/pkg/wrapper/controllerservice.go#LL41C2-L41C29

Reply:
https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/volumes/csi-wrapper/pkg/wrapper/controllerservice.go#L85